examples/CSharp/Tutorial 1.1 - Hello Window/Program.cs

```diff
@@ -12,7 +13,7 @@ private static void Main(string[] args)
         {
             //Create a window.
             var options = WindowOptions.Default;
-            options.Size = new Size(800, 600);
+            options.Size = new Vector2D<int>(800, 600);
```

Comment:
90%  sure this redundant.
Consider 
```suggestion
            options.Size = new Vector2D(800, 600);
```
(make sure to also change this in the others)

Reply:
You may find yourself in the 10% my friend.

Reply:
Well I didn't realize that'd never be possible because of the static types. Kinda sucks but idc